tests/legacy-cli/e2e/initialize/300-log-environment.ts

```diff
@@ -14,6 +15,10 @@ export default async function () {
     console.log(`  ${envName}: ${process.env[envName]!.replace(/[\n\r]+/g, '\n        ')}`);
   });
 
-  await exec('which', 'ng', getActivePackageManager());
+  // On Windows, `which` might not be available.
+  if (!isWindowsTestMode()) {
```

Comment:
We should log here that we didn't check it in windows environments, so that we have a note of that information in the outputs.